src/DotNet/Resources/ResourceBinaryWriter.cs

```diff
@@ -0,0 +1,27 @@
+using System.IO;
+
+namespace dnlib.DotNet.Resources {
+	/// <summary>
+	/// Extension of <see cref="BinaryWriter"/> for writing resource set elements
+	/// </summary>
+	public sealed class ResourceBinaryWriter : BinaryWriter {
+		/// <summary>
+		/// Format version of the resource set
+		/// </summary>
+		public int FormatVersion { get; internal set; }
+
+		/// <summary>
+		/// Specifies the target reader type of the resource set
+		/// </summary>
+		public ResourceReaderType ReaderType { get; internal set; }
+
+		/// <inheritdoc />
+		public ResourceBinaryWriter(Stream stream) : base(stream) { }
```

Comment:
This ctor is public but the properties above only has internal setters. Seems like the ctor should be internal.